tests/test_documentation_hygiene.py

```
@@ -9,12 +9,18 @@
 _PRODUCT_GAP_BASELINE = _ROOT / "docs" / "product-technical-gap-baseline.md"
+_PRODUCT_REQUIREMENTS = _ROOT / "docs" / "product-requirements.md"
 _ROLE_CATALOG_COLUMNS = (
     "cataloged_team_id",
     "cataloged_corporate_entity_id",
     "cataloged_person_id",
 )
 _ADR_NAME = re.compile(r"^(?P<number>[0-9]{4})-.+\.md$")
+_PRD_REQUIREMENT_HEADING = re.compile(r"^### (?P<identifier>PRD-FR-[0-9A-Z-]+)\b", re.MULTILINE)
+_PRD_ADR_CLAUSE = re.compile(r"\bADRs?\s+(?P<references>[^.;)]*)")
+_ADR_NUMBER_OR_RANGE = re.compile(
+    r"(?P<start>[0-9]{4})(?:\s*[–-]\s*(?P<end>[0-9]{4}))?"
```

Ruff (0.16.3)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)